diff --git a/layers/+readers/pdf/packages.el b/layers/+readers/pdf/packages.el
index 59cf0be1b..cb75442bf 100644
--- a/layers/+readers/pdf/packages.el
+++ b/layers/+readers/pdf/packages.el
@@ -20,6 +20,17 @@
     (progn
       (pdf-tools-install)
 
+      (setq-default pdf-view-display-size 'fit-page
+                    pdf-view-midnight-colors `(,(face-attribute 'default :foreground) .
+                                               ,(face-attribute 'default :background)))
+;;      (add-hook 'pdf-view-mode-hook
+;;                (lambda ()
+;;                  (setq-default pdf-view-midnight-colors `(,(face-attribute 'default :foreground) .
+;;                                                           ,(face-attribute 'default :background)))))
+;;      (when (eq frame-background-mode 'dark)
+      (add-hook 'pdf-view-mode-hook
+                'pdf-view-midnight-minor-mode)
+
       (spacemacs|define-transient-state pdf-tools
         :title "PDF-tools transient state"
         :on-enter (setq which-key-inhibit t)
